src/locale/pl/dictionary.ftl

```diff
@@ -0,0 +1,1950 @@
+
+search-find-content = Szukaj zasobów
+
+search-search = Search
+  [html/placeholder]  Szukaj
+
+search-or = lub
+
+search-advanced-search = Zaawansowane wyszukiwanie
+
+# SEARCH - Content - src/scripts/pages/browse-content/browse-content-template.html
+
+search-header = Witaj w cyfrowej bibliotece OpenStax CNX
+
+search-content =
+  | Zasoby OpenStax CNX udostępniane są w dwóch postaciach:
+  | <strong>modułów/stron</strong>, stanowiących małe "porcje wiedzy," oraz
+  | <strong>kolekcji/książek</strong>, stanowiących zestawy modułów ułożone w
+  | formę podręcznika, kursu lub publikacji o innym przeznaczeniu. Otwarte
+  | licencje pozwalają na swobodne użytkowanie wszystkich zasobów.
+
+search-pages = Strony: { $count }
+
+search-books = Książki: { $count }
+
+
+
+### ADVANCED SEARCH ###
+# Link: http://cnx.org/search
+# Content related to the advanced search form.
+
+# ADVANCED SEARCH - Page Title - src/scripts/modules/search/advanced/advanced.coffee
+
+advanced-search-pageTitle = Zaawansowane wyszukiwanie
+
+# ADVANCED SEARCH - Minimal Search Results Table Partial - src/scripts/modules/minimal/search/results/list/table-partial.html
+
+minimal-search-results-table-partial-books = Książki
+
+minimal-search-results-table-partial-pages = Strony
+
+minimal-search-results-table-partial-miscellaneous = Dodatkowe
+
+# ADVANCED SEARCH - Advanced Search Minimal Page Title - src/scripts/modules/minimal/search/advanced/advanced.coffee
+
+advanced-search-minimal-pageTitle = Zaawansowane wyszukiwanie
+
+# ADVANCED SEARCH - Search Minimal Page Title - src/scripts/modules/minimal/search/advanced/advanced.coffee
+
+search-minimal-pageTitle = Szukaj
+
+# ADVANCED SEARCH - Content - src/scripts/modules/search/advanced/advanced-template.html
+
+advanced-search-author = Autor
+
+advanced-search-title = Tytuł
+
+advanced-search-subject = Dziedzina
+
+advanced-search-subject-default-any = Dowolna
+
+advanced-search-subject-arts = Muzyka i plastyka
+
+advanced-search-subject-business = Przedsiębiorczość
+
+advanced-search-subject-humanities = Nauki humanistyczne
+
+advanced-search-subject-math = Matematyka i statystyka
+
+advanced-search-subject-science = Pozostałe nauki ścisłe, przyrodnicze i techniczne
+
+advanced-search-subject-social-sciences = Nauki społeczne
+
+advanced-search-keywords = Słowa kluczowe
+
+advanced-search-type = Typ
+
+advanced-search-type-default-any = Dowolny
+
+advanced-search-type-books = Książki
+
+advanced-search-type-pages = Strony
+
+advanced-search-language = Język
+
+advanced-search-language-default-any = Dowolny
+
+advanced-search-publication-date = Rok publikacji
+
+advanced-search-publication-date-default-any = Dowolny
+
+advanced-search-sort = Sortuj według
+
+advanced-search-sort-default-relevance = trafności
+
+advanced-search-sort-publication-date = Roku publikacji
+
+advanced-search-sort-popularity = popularności
+
+advanced-search-submit = Szukaj
+
+advanced-search-language-option = { $native } ({ $code ->
```

Comment:
This summer, we had an intern modify this dropdown to display the actual languages we have content for rather than this list of all possibilities. Since this list is coming from the database now, how will that change your implementation?